plone/app/drafts/proxy.py

```diff
@@ -1,15 +1,13 @@
 # -*- coding: utf-8 -*-
+from collections import MutableMapping
+
 from Acquisition import aq_base
 from plone.app.drafts.interfaces import IDraftProxy
-from UserDict import DictMixin
 from zope.annotation.interfaces import IAnnotations
 from zope.component import adapter
-from zope.interface import implementedBy
-from zope.interface import implementer
-from zope.interface import providedBy
-from zope.interface.declarations import getObjectSpecification
-from zope.interface.declarations import ObjectSpecificationDescriptor
-
+from zope.interface import implementedBy, implementer, providedBy
+from zope.interface.declarations import (ObjectSpecificationDescriptor,
+                                         getObjectSpecification)
```

Comment:
Why are you combining the imports? Plone style is to have a separate import for each name to avoid noisy diffs.

Reply:
my bad, I ran isort on it but it didn't have isort configuration in setup.cfg--added.